react/src/component/login/email.tsx

```diff
@@ -22,20 +22,21 @@ export const LoginEmailConfirm = (props) => {
             headers: {
                 "Content-Type": "application/json"
             }
-        }).then(() => 
+        }).then(() =>
         updateLoginStatusRef(loginStatusRef, appWrapper)
-        .then(() => navigate(`/react/${appName}`))
+        .then(() => navigate(`/${appName}`))
         )
     }
+
     return (
         <Container>
             {
-                needCompleteRegistration ? (
+                needCompleteRegistration === '1' ? (
```

Comment:
This is going to get us in trouble. Is there no utility module for evaluating booleans from the schema?